apps/daimo-mobile/src/logic/key.ts

```diff
@@ -0,0 +1,109 @@
+import { isDERPubKey, assert, parseAndNormalizeSig } from "@daimo/common";
+import { daimoAccountABI } from "@daimo/contract";
+import * as ExpoEnclave from "@daimo/expo-enclave";
+import { SigningCallback } from "@daimo/userop";
+import { base64urlnopad } from "@scure/base";
+import * as Crypto from "expo-crypto";
+import {
+  Hex,
+  bytesToHex,
+  concat,
+  encodeAbiParameters,
+  getAbiItem,
+  hexToBytes,
+  isHex,
+} from "viem";
+
+import { env } from "./env";
+import { Log } from "./log";
+
+export function parseAddDeviceString(addString: string): Hex {
+  const [prefix, pubKey] = addString.split(":");
+  assert(prefix === "addkey");
+  assert(isHex(pubKey));
+  assert(isDERPubKey(pubKey));
+  return pubKey;
+}
+
+export function createAddDeviceString(pubKey: Hex): string {
+  assert(isDERPubKey(pubKey));
+  return `addkey:${pubKey}`;
+}
+
+export function getWrappedRawSigner(
```

Comment:
super clean

i'd ad a short comment here

```
// Creates minimal Webauthn signatures using a hardware enclave key.
// Unlike passkeys, which are backed up, enclave keys never leave the device.
```

and above
```
// Creates a custom URI for the Add Device QR code.
...
// Parses the custom URI from the Add Device QR code.
```

Reply:
done